paasta_tools/marathon_tools.py

```diff
@@ -89,7 +93,75 @@
 logging.getLogger('marathon').setLevel(logging.WARNING)
 
 MarathonServers = namedtuple('MarathonServers', ['current', 'previous'])
```

Comment:
Add type annotations for MarathonServer so mypy knows these are MarathonConfig objects.